extensions/amp-browsi/0.1/test/validator-amp-browsi.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>amp-browsi example</title>
+  <link rel="canonical" href="amps.html">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>
+    body {
+      -webkit-animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+      -moz-animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+      -ms-animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+      animation: -amp-start 8s steps(1, end) 0s 1 normal both
+    }
+    @-webkit-keyframes -amp-start {
+      from {visibility: hidden}
+      to {visibility: visible}
+    }
+
+    @-moz-keyframes -amp-start {
+      from {visibility: hidden}
+      to {visibility: visible}
+    }
+
+    @-ms-keyframes -amp-start {
+      from {visibility: hidden}
+      to {visibility: visible}
+    }
+
+    @-o-keyframes -amp-start {
+      from {visibility: hidden}
+      to {visibility: visible}
+    }
+
+    @keyframes -amp-start {
+      from {visibility: hidden}
+      to {visibility: visible}
+    }</style>
+  <noscript>
+    <style amp-boilerplate>body {
+      -webkit-animation: none;
+      -moz-animation: none;
+      -ms-animation: none;
+      animation: none
+    }</style>
+  </noscript>
+  <style amp-custom>
+    amp-browsi {
+      color: red;
+    }
+  </style>
+  <script async custom-element="amp-browsi" src="https://cdn.ampproject.org/v0/amp-browsi-0.1.js"></script>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+<amp-browsi layout="responsive" width="150" height="80"></amp-browsi>
```

Comment:
Please add a `<!-- Valid --->` above this line showing this line is testing a valid usage of this component.

Reply:
Done, tnx.